cli/commands/build.js

```diff
@@ -28,38 +31,40 @@ const mkdir = directory => {
 function getDestinationFromConfig(source, target) {
   const config = packageConfig.getPackageConfig();
   const buildConfig = config.build;
+  const buildDirectory = buildConfig.directory;
 
-  if (!buildConfig) {
+  if (!buildDirectory) {
     throw new Error(
-      'No build configuration has been found. It is a "[build]" section on you "clio.toml" file.'
+      'The build directory is missing on your "clio.toml".\n\nExample:\n\n[build]\ndirectory = "build"\n'
```

Comment:
I notice that you use single quotes. Are you using Prettier locally? Or is it because you have nested quotes?

Reply:
It's because there are nested quotes on`"clio.toml`". I run prettier every time I save. :)

Reply:
Ahh okay, perfect! :)